vendor-extra/JatsContentBundle/src/EventListener/BuildView/SectionListener.php

```diff
@@ -54,7 +55,7 @@ protected function handle(Element $object, TemplateView $view) : TemplateView
         return $view->withArgument(
             'content',
             array_map(
-                function (NonDocumentTypeChildNode $child) use ($childContext) : TemplateView {
+                function (NonDocumentTypeChildNode $child) use ($childContext) : View {
```

Comment:
https://github.com/libero/browser/blob/master/vendor-extra/ViewsBundle/src/Views/CallbackViewConverter.php#L19 returns `View` so this matches that.